NEWS.md

```diff
@@ -12,6 +12,7 @@
 
 ### Bug fixes
 * Fix ordering of electronic access items for inventory-hierarchy, oai-pmh-view ([MODINVSTOR-1224](https://folio-org.atlassian.net/browse/MODINVSTOR-1224))
+* Errors during save phase of marc-migrations (instances) ([MODINVSTOR-1389](https://folio-org.atlassian.net/browse/MODINVSTOR-1389))
```

Comment:
The mod-inventory-storage micro service doesn't know anything about marc migration. Please specify in news (and in following squash merge commit) what exactly changed in this PR (what endpoint logic was modified and how)